src/builder.rs

```diff
@@ -325,6 +327,8 @@ impl<'req, 'dev, 'win, W: HasRawWindowHandle> PixelsBuilder<'req, 'dev, 'win, W>
         let mut pixels = Vec::with_capacity(pixels_buffer_size);
         pixels.resize_with(pixels_buffer_size, Default::default);
 
+        let alpha_mode = surface.get_supported_alpha_modes(&adapter)[0];
```

Comment:
Not sure if this is how you'd want to do it, but I found some examples of people doing this rather than just using "auto".

Reply:
This seems ok to me. The docs claim this method returns at least one element. So this should never panic.